package.json

```diff
@@ -19,6 +16,7 @@
 		"angular-mocks": "1.5.0",
 		"jasmine-jquery": ">=2.0.0",
 
+		"grunt": "^0.4.1",
```

Comment:
If you split only this change (and the `main` change) into another PR, I'll merge that. The rest of this PR is pending for a better solution (check my other comment).